File: Day345/RedistributeCharacterstoMakeAllStringsEqual.go
func makeEqual(words []string) bool {
    l := len(words)
    arr := make([]int, 26)
    for _,v := range words {
        for _,c := range v {
            arr[c-'a']++
        }
    }
    for _,v := range arr {
        if v % l != 0 {
            return false
        }
    }
    return true
}

File: Day345/RedistributeCharacterstoMakeAllStringsEqual.java
class Solution {
    public boolean makeEqual(String[] words) {
        int arr[] = new int[26];
        int n = words.length;
        for (String w: words) {
            for (char c: w.toCharArray()) {
                arr[c-'a']++;
            }
        }
        for (int i=0; i<26; i++) {
            if (arr[i]%n!=0){
                return false;
            }
        }
        return true;
    }
}